web/__init__.py

```diff
@@ -98,6 +134,20 @@ def app_api_version():
     }
 
 
+@app.post('/api/web/config/upload')
+def app_api_web_config_upload():
+    if request.method != 'POST':
+        return web.util.flash_redirect()
+    if 'login_file' not in request.files:
```

Comment:
Please add a blank line after "return" statements - it makes the structure easier to follow.

Reply:
:+1: